downstream/assemblies/central-auth/assembly-assign-hub-admin-permissions.adoc

```diff
@@ -9,10 +9,10 @@ Hub administrative users will need to be assigned the role of _hubadmin_ in orde
 
 .Procedure
 . Navigate to the `ansible-automation-platform` realm on your SSO client.
-. Under the _Manage_ section on the side navigation bar, click *Users*.
+. Under the _Manage_ section on the side navigation bar, click btn:[Users].
```

Comment:
I believe this is actually a menu item and not a button. It also violates accessibility by using a directional statement "side navigation". I suggest rewriting to something like the following:

From the navigation bars, select menu:Manage[Users].